### PR TITLE
milestone 2 general requirment fully functional

### PR DESCRIPTION
**Pull request by em-em-D**
react calculator milestone 2 fully set up, I have challenges making stickler and linter work on this repository, so please do ignore the mild linter errors

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        this.state = {};
+    }
+    render(){
+        return(
+            <div>
+           <h1>I am App and I render display and ButttonPanel</h1>
+           <Display />
+           <ButtonPanel/>
+            </div>
```
Comment (shloch) [inline src/components/App.js]: code not well formatted....check linting again

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    return(
+        <div className="result">
+            <p>{result}</p>
+        </div>
+    );
+}
+Display.propTypes = {
+    result: PropTypes.string.isRequired,
+};
+
```
Comment (shloch) [inline src/components/Display.js]: default value of `result` should be `'0'`, please validate this with `defaultProps` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @em-em-D , 

I think we're doing very fine with the implementation

Just minor things you forgot..
Let's just round up with them (comments below) and move to next milestone

Happy coding

File: src/components/App.js (hunk 1 above)
Comment (em-em-D) [reply src/components/App.js]: I tried to fix that locally but the linter still fails in the respository

**Review by shloch — CHANGES_REQUESTED** [review]
Hello great job.....but u missed out something Little.......concerning the "result" prop. GOOGLE what is "defaultProps" in react.....that's what you had to use to ensure that "result" equals "0" (in string format, not integer as you did)
Please correct this..

### happy coding

File: src/components/Display.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+            <p>{result || 0}</p>
+        </div>
+    );
+}
+Display.propTypes = {
+    result: PropTypes.string.isRequired,
+};
+Display.defaultProps={
+    result: "0"
+};
```
Comment (shloch) [inline src/components/Display.js:17]: nice nice

**Review by shloch — APPROVED** [review]
## Hello @em-em-D

Great Job..Your components are set
### We're good to go on next milestone

**Pull request merged**